goss-testing/scripts/postgresql_backups_check.sh

```diff
@@ -44,6 +44,14 @@ do
     esac
 done
 
+tmp_dir=$(mktemp -d)
+trap 'rm -rf "${tmp_dir}"' EXIT
+admin_secret=$(kubectl get secrets admin-client-auth -ojsonpath='{.data.client-secret}' | base64 -d)
+curl -k -s -d grant_type=client_credentials \
+        -d client_id=admin-client \
+        -d client_secret="$admin_secret" https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token > "${tmp_dir}/cray-token.json"
+export CRAY_CREDENTIALS="${tmp_dir}/cray-token.json"
```

Comment:
At the end of the test, should the tmp dir be deleted and the CRAY_CREDENTIALS be unset?

Reply:
The dir is deleted (there's `trap` statement at line 48). I also added `unset CRAY_CREDENTIALS` to it.